content/docs/glossary/muxed-accounts.mdx

```diff
@@ -0,0 +1,201 @@
+
+
+## Supported Operations
+
+Not all operations can be used with muxed accounts. For example, you cannot set the destination of a [`CreateAccount`](../start/list-of-operations.mdx#create-account) operation to be a muxed account, because only their shared, underlying `G...` account exists in the ledger. However, you can use them with:
+
+  - the source account of *any* operation or [transaction](./transactions.mdx),
+  - the destination of all three types of payments:
+    * [`Payment`](../start/list-of-operations.mdx#payment),
+    * [`PathPaymentStrictSend`](../start/list-of-operations.mdx#path-payment-strict-send), and
+    * [`PathPaymentStrictReceive`](../start/list-of-operations.mdx#path-payment-strict-receive)
+  - the destination of an [`AccountMerge`](../start/list-of-operations.mdx#account-merge), and
+  - the fee source of a [fee-bump transaction](./fee-bumps.mdx).
+
+We'll demonstrate some of these later in the [Examples](#examples).
+
+
+## Horizon Support
+
+Even though at the end of the day, only the underlying `G...` account _truly_ exists in the Stellar ledger, the [Horizon API](../../api/introduction/) will make some effort to interpret and track the muxed accounts responsible for certain actions.
+
+**TODO.**
+
+
+## Frequently Asked Questions
+**TODO.**
+
+### How do I support it in my wallet?
+### Should I support Muxed Accounts in my wallet?
+### etc.
+
+
+## Examples
+
+In this section, we'll demonstrate how to create muxed accounts and how they interact with some of the supported operations. To drive home the fact that custodial account workarounds based on transaction memos are superfluous now, we'll use that as a skeleton for our example structure.
+
+
+### Preamble
+First, let's create two accounts and then a handful of virtual accounts representing "custodial customers" that the parent account manages.
+
+<CodeExample title="Preamble">
+
+```js
+const sdk = require("stellar-sdk");
+
+const passphrase = "Test SDF Network ; September 2015";
+const url = "https://horizon-testnet.stellar.org";
+let server = new sdk.Server(url);
+
+const custodian = sdk.Keypair.fromSecret("SAQLZCQA6AYUXK6JSKVPJ2MZ5K5IIABJOEQIG4RVBHX4PG2KMRKWXCHJ");
+const outsider  = sdk.Keypair.fromSecret("SAAY2H7SANIS3JLFBFPLJRTYNLUYH4UTROIKRVFI4FEYV4LDW5Y7HDZ4");
+
+async function preamble() {
+  custodianAcc = await server.loadAccount(custodian.publicKey());
+  outsiderAcc  = await server.loadAccount(outsider.publicKey());
+
+  customers = ["1", "22", "333", "4444"].map(
+    (id) => custodianAcc.createSubaccount(id)
```

Comment:
Is this the preferred way of creating a muxed account (and not `new MuxedAccount()`)?

Reply:
+1 on this question. I remember some discussion on this when implementing support in the Lab or Account Viewer. Are there differences/value  in the use of either method?

Reply:
Good question! Suppose `account` is some Stellar account loaded from Horizon. Then, these two are functionally equivalent:

```js
let muxed1 = new MuxedAcccount(account, "1");
let muxed2 = account.createSubaccount("1");
```

These are also functionally equivalent (in situations where you only have a muxed account variable around):

```js
let muxed3 = new MuxedAccount(muxed1.baseAccount(), "3");
let muxed4 = muxed1.createSubaccount("3");
```

The `createSubaccount()` method was created with this latter scenario in mind: conveniently create "sibling" muxed accounts from existing ones. It was added to regular `Account`s for interface compatibility (though the [naming has some contention](https://github.com/stellar/js-stellar-base/issues/433)). 

-------------------

Given that this question came up, maybe this is confusing? Perhaps having two ways to do the same thing is a bad move; the [Zen of Python](https://www.python.org/dev/peps/pep-0020/) comes to mind here:  

> There should be one-- and preferably only one --obvious way to do it.

Thoughts on dropping this `createSubaccount()` helper, then?